docs/tutorials/IndexerMigration.md

```diff
@@ -0,0 +1,132 @@
+## Migrating from Indexer to Conduit
+
+The [Algorand Indexer](https://github.com/algorand/indexer) provides both a block processing pipeline to ingest block
+data from an Algorand node into a Postgresql database, and a rest API which serves that data.
+
+The [Conduit](https://github.com/algorand/indexer/blob/develop/docs/Conduit.md) project provides a modular pipeline
+system allowing users to construct block processing pipelines for a variety of use cases as opposed to the single,
+bespoke Indexer construction.
+
+### Migration
+Talking about a migration from Indexer to Conduit is in some ways difficult because they only have partial overlap in
+their applications. For example, Conduit does _not_ currently include a rest API either for checking pipeline health
+or for serving data from the pipeline. 
+
+Here is the Indexer architecture diagram at a high level. The raw block data is enriched by the account data retrieved
+from the local ledger, and everything is written to Postgresql which can then be queried via the API.
+```mermaid
+graph LR;
+    algod["Algod"]
+    index["Indexer"]
+    ledger["Local Ledger"]
+    psql["Postgresql"]
+    restapi["Rest API"]
+    
+    algod-->index;
+    subgraph "Data Pipeline"
+        index-->ledger;
+        ledger-->index;
+        index-->psql;
+    end
+    psql-->restapi;
+    restapi-->psql;
+```
+
+However, Conduit was built to generalize and modularize a lot of the tasks which Indexer does when ingesting block data
+into its database. For that reason you can swap out the core data pipeline in Indexer with an equivalent Conduit
+pipeline--and that's just what we've done!
+
+```mermaid
+graph LR;
+    algod["Algod"]
+    be["block_evaluator Processor"]
```

Comment:
are we going to have follow up PRs to remove block_evaluator contents from these docs?

Reply:
Thanks, I just removed references to the block evaluator.